src/login/lobby.cpp

```diff
@@ -779,42 +781,78 @@ int32 lobbyview_parse(int32 fd)
                     std::memset(CharName, 0, sizeof(CharName));
                     std::memcpy(CharName, sessions[fd]->rdata.data() + 32, sizeof(CharName));
 
-                    // find assigns
-                    const char* fmtQuery = "SELECT charname FROM chars WHERE charname LIKE '%s'";
+                    // Sanitize name
+                    char escapedCharName[16 * 2 + 1];
```

Comment:
I may have missed it, but I'm not seeing any place where we check charName length for validation.

Reply:
Its handled in the client, but I added some checks for it in case people get sneaky